### DISCUSSION TITLE
My Results are awlays better in Prophet when compared to Orbit's DLT

### DISCUSSION THREAD

**Discussion by ssampathkumar104** [uber/orbit#870] [General]
Hi Everyone,

I am a beginner learning data science and as part of my colleage project. I have choosen to compare Prophet & Orbit time series forecasting. As per the theory I assumed that Orbit's forecast will yield better results compared to Prophet but I always ended-up having better results in Prophet. 

Please find my model below and could anyone help me improve Orbit's model such that I can prove DLT is better for time series forecasting than Prophet.

**Prophet model:**
```
self.model = Prophet(interval_width=0.95,
                                 growth='linear',
                                 yearly_seasonality=True,
                                 weekly_seasonality=True,
                                 changepoint_range=0.95,
                                 changepoint_prior_scale=0.1,
                                 holidays=self.holiday_df,
                                 holidays_mode='additive',
                                 seasonality_mode='additive',
                                 seasonality_prior_scale=1.01)
```

**Orbit's model:**
```
self.model = DLT(response_col='y',
                             date_col='ds',
                             seasonality=182,
                             estimator='stan-map',
                             global_trend_option='loglinear',
                             damped_factor=0.7,
                             regressor_col=self.regressor_cols)
```

Thanks in advance

**Comment by edwinnglabs** [comment]
@ssampathkumar104 its hard to tell in your use case unless you can provide some more details or some example data.
